client/blocks/color-scheme-picker/README.md

```diff
@@ -0,0 +1,63 @@
+Color Scheme Picker
+================
+
+This component renders a Color Scheme Picker.
+
+When using the component without any props, any selection will immediately be saved to the `colorSchemes` property in the `calypso_settings`.
+When the props `temporarySelection` and `onSelection` are provided, a selection will only be set locally and the provided onSelection handler called (e.g. to save the selection at a later stage via a save button).
+
+
+## Usage
+
+```es6
```

Comment:
Tiny nit that I missed the first time around - could we change this to `jsx` please?
That'll give slightly better syntax highlighting in GH :)